deploy/helm/sumologic/values.yaml

```diff
@@ -29,7 +29,7 @@ sumologic:
   # Collector name
   #collectorName: ""
 
-  # Cluster name
+  # Cluster name: Note spaces are not allowed and will be replaced with dashes.
   clusterName: "kubernetes"
```

Comment:
We would have to change `clusterName` to `clusterNameReplaceSpaceWithDash` here, however this is "technically" a breaking change and would have to be included with 1.0. Otherwise, we can leave the name of this config the same, and just include the comment above it specifying this behavior, since up to now having a space in cluster name would have failed anyway. What do you think?

Reply:
I think we can keep the name same `clusterName` and just specify the comment.

Reply:
We would not have to change it actually because I created `clusterNameReplaceSpaceWithDash` as a function in the helpers.tpl file.  If customers use a space before it would work in the collection, but would break in the explore UI at which point we would tell them change it to have no spaces.  So its unlikely anyone has a space given it breaks the Explore tab.

Reply:
That is why I think it is ok to do this in 1.1 as opposed to 1.0.

Reply:
Aha, totally missed that helper file. Thanks Frank